README.md

```diff
@@ -107,292 +107,73 @@ Following the installation documentation below, you'll be able to run your own i
 
 # Installation #
 
-## Dependencies ##
+## Debian dependencies ##
```

Comment:
Writing Debian dependencies excludes even before read other users. The original formulation allow other distribution OS not to feel excluded.